neo/Consensus/ConsensusService.cs

```diff
@@ -104,9 +117,13 @@ private void CheckSignatures()
         private void FillContext()
         {
             IEnumerable<Transaction> mem_pool = Blockchain.Singleton.GetMemoryPool();
+
             foreach (IPolicyPlugin plugin in Plugin.Policies)
                 mem_pool = plugin.FilterForBlock(mem_pool);
             List<Transaction> transactions = mem_pool.ToList();
+	    //Maybe there is an attach here, the Primary can pick up the largest/worse TXs available in order to prejudicate Backups behavior and processing time
```

Comment:
What's this mean?

Reply:
The implementation is incomplete, Erik, it will still take some weeks/months.
This is the place in which the Primary could filter the mempool with specific criteria, right?
It is just to remember that an additional function on the Plugin could be created in order to ponder that point I mentioned above related to the Knapsack Problem. However, this is an additional feature that will probably not be handled here, only if everyone's agree and we really need such feature in massive number of transactions.

Reply:
Vitor means that speaker can select a single tx to forge small block intentionally. However, we could desl with this in two ways: 1. informing the community of average block.sizes per consensus node; 2. denying to.sign a block that is "too diferent" from the locally available mempool